demonstrations/tutorial_tn_basics.metadata.json

```
@@ -0,0 +1,49 @@
+{
+    "title": "Tutorial: ABC of tensor networks",
+    "authors": [
+        {
+            "id": "emiliano_godinez"
+        },
+        {
+            "id": "ahmed_darwish"
```

Please swap this out like this:
```suggestion
            "username": "emiliano"
        },
        {
            "username": "ShiroRaven"
```
Also, @Shiro-Raven , you're still missing a picture and the _Headline_ on your account.

---

is @Shiro-Raven still an author of the demo?

---

Hey there! Very sorry for the radio silence. Emiliano and I actually agreed a long time ago that he will go on and have complete authorship of this demo, so please remove me from the metadata and the authors section